### PR TITLE
Moved pagination from admin to be used in react vapor

### PR DESCRIPTION
**Pull request by sroy3**

### CODE REVIEW

File: scss/guide.scss [hunk 1]
```diff
@@ -45,6 +45,7 @@
   @import 'tables/table-actions';
   @import 'tables/table-align-header';
   @import 'tables/table-collapsible-rows';
+  @import 'tables/table-navigation';
```
Comment (GermainBergeron) [inline scss/guide.scss]: I think we should name the file table-pagination or simply pagination (it's not always used in a table)

File: docs/_components/tables/05_table-navigation.html [hunk 2]
```diff
@@ -0,0 +1,100 @@
+---
+type: tables
+title: Table navigation
+usage: Adds buttons to change the number of items per page and change page.
+stylesheet: '/scss/tables/table-collapsible-rows.scss'
```
Comment (GermainBergeron) [inline docs/_components/tables/05_table-navigation.html]: wrong stylesheet

File: scss/guide.scss (hunk 1 above)
Comment (sroy3) [inline scss/guide.scss]: Done

File: docs/_components/tables/05_table-navigation.html (hunk 2 above)
Comment (sroy3) [inline docs/_components/tables/05_table-navigation.html]: Oops, changed.

**Pull request merged**